src/database.ts

```diff
@@ -1665,7 +1895,7 @@ class Database extends ServiceObject {
    *
    * const table = database.table('Singers');
    */
-  table(name) {
+  table(name: string): Table {
```

Comment:
```suggestion
  table(name: string) {
```
Same as above, inferred form `return` statement